guides/micronaut-google-cloud-http-function/micronaut-google-cloud-http-function.adoc

```diff
@@ -112,9 +112,12 @@ To run the function locally:
 [source, bash]
 ----
 curl -w "\n" localhost:8080/micronautguide
-Example Response
 ----
 
+```
```

Comment:
I am not sure that this is proper asciidoc. I think they are Github fenced code blocks